integration-tests/tests/run-workflow.js

```diff
@@ -50,23 +58,28 @@ const testRunWorkflowFn = _.flow(
   ])
   // stopLoggingPageAjaxResponses()
 
+  const disablePageLogging = logPageConsoleMessages(page)
+  const start = Date.now()
   await pRetry(async () => {
     try {
       await findInGrid(page, 'Succeeded', { timeout: 65 * 1000 }) // long enough for the submission details to refresh
     } catch (e) {
+      try {
+        await findInGrid(page, 'Running', { timeout: 1000 })
+        console.info(`Workflow is running, elapsed time (minutes): ${(Date.now() - start) / (1000 * 60)}`)
+      } catch (e) {
+        console.info(`Workflow not yet running, elapsed time (minutes): ${(Date.now() - start) / (1000 * 60)}`)
+      }
       throw new Error(e)
     }
-  }, { retries: 10, factor: 1 })
-
-  await click(page, navChild('data'))
-  await click(page, clickable({ textContains: 'test_entity' }))
-  await findInDataTableRow(page, testEntity.name, testEntity.attributes.input)
-})
+  }, { retries: 15, factor: 1 })
+  disablePageLogging()
```

Comment:
It's not immediately clear to me why we're storing this function in a value and then calling it - does `page` from line 61 go out of scope?

Reply:
`logPageConsoleMessages` enables logging and returns a function that you can use to turn it off later. At least that's what the code looks like to me.